src/jit/morph.cpp

```diff
@@ -3587,7 +3658,7 @@ GenTreeCall* Compiler::fgMorphArgs(GenTreeCall* callNode)
             if (!isBackFilled)
             {
 #if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-                if (isStructArg)
+                if (isStructArg) 
```

Comment:
What is this change and the one below? Just want to make sure this isn't changing the line endings.